src/assets/icons/chevron-down.svg

```diff
@@ -0,0 +1 @@
+<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><path transform="rotate(90 12 12)" d="M17.192 11.778L9.414 4 8 5.414l6.364 6.364L8 18.142l1.414 1.414z"/></svg>
```

Comment:
as this is only a rotated version of chevron. It would be nice if instead of using a new file, we could come up with a good solution to reuse files we already have
see #49